Themis/ViewModels/Assessment/Text/TextAssessmentViewModel.swift

```diff
@@ -51,17 +51,17 @@ class TextAssessmentViewModel: AssessmentViewModel {
             let submissionService = SubmissionServiceFactory.service(for: exercise)
             // We are not actually getting the submission for assessment, but this is the only endpoint that can be used to
             // get the submission based on the submissionId without locking it
-            let response = try await submissionService.getSubmissionForAssessment(submissionId: submissionId)
+            let response = try await submissionService.getSubmissionForAssessment(submissionId: submissionId, correctionRound: 0)
```

Comment:
Wouldn't it make more sense to change the signature of the method to use CorrectionRound instead of an integer?
I think this might be more readable than to know that 0 stands for the first correction round.
What do you think?

```suggestion
            let response = try await submissionService.getSubmissionForAssessment(submissionId: submissionId, correctionRound: .first)
```

Reply:
I agree. This would improve the readability 👍